packages/extension-ui/src/components/Address.tsx

```diff
@@ -201,13 +201,15 @@ function Address ({ actions, address, children, className, genesisHash, isExtern
                 title={t('copy address')}
               />
             </CopyToClipboard>
-            <FontAwesomeIcon
-              className={ isHidden ? 'hiddenIcon' : 'visibleIcon'}
-              icon={isHidden ? faEyeSlash : faEye}
-              onClick={_toggleVisibility}
-              size='sm'
-              title={t('account visibility')}
-            />
+            {actions && (
+              <FontAwesomeIcon
+                className={ isHidden ? 'hiddenIcon' : 'visibleIcon'}
```

Comment:
```suggestion
                className={isHidden ? 'hiddenIcon' : 'visibleIcon'}
```

throw-away spacing comment.